modules/st2_systemd/manifests/startup.pp

```diff
@@ -0,0 +1,30 @@
+# Definition: st2_systemd
+#
+#  Lets create some Systemd scripts
+#
+define st2_systemd::startup (
```

Comment:
This feels disjointed. The `ng_init` flag in the upstream repo explicitly flags whether to use these init scripts or not.

I get we're moving fast, so help me visualize what this looks like? No `ng_init` attribute on base st2 class, and then this if RHEL?